README.md

```diff
@@ -19,7 +19,50 @@ This repository allows you to quickly install Apache Solr for Drupal 9+ into a [
      * Set "solr core" to `dev`
      * Under "Advanced server configuration" set the "solr.install.dir" to `/opt/solr`.
 
-1. `ddev restart`
+6. `ddev restart`
+
+## Installation on Silverstripe 4+
+
+1. `ddev get ddev/ddev-drupal9-solr && ddev restart`
+2. Install the required/relevant Silverstripe requirements: `ddev composer require firesphere/solr-search`
+    * Note that there are currently some tricky dependencies. This is known and to be fixed soon (tm)
+4. Solr is set to use the FileConfigStore. Ensure it's data location is at `.ddev/solr` and the host is set to `solr`:
+```yml
+---
+Name: LocalSolrSearch
+After:
+  - 'SolrSearch'
+Only:
+  environment: 'dev'
+---
+Firesphere\SolrSearch\Services\SolrCoreService:
+  config:
+    endpoint:
+      localhost:
+        host: solr
+  store:
+    path: './.ddev/solr'
+Firesphere\SolrSearch\Indexes\BaseIndex:
+  dev:
+    Classes:
+      - Page
+    FulltextFields:
+      - Title
+      - Content
+```
+5. Edit `.ddev/docker-compose.solr.yml` and on line 60, change `- ./solr:/solr-conf` to `- ./solr/dev:/solr-conf`
+    * Historically, Silverstripe uses the name of the core as sub-folder for the location of the core, which is why the location in the YML needs updating.
+    * [Refer to the documentation of the Silverstripe Solr module](https://firesphere.github.io/solr-docs/) for how to configure it.
+6. Run the Silverstripe Solr configuration command `ddev exec vendor/bin/sake dev/tasks/SolrConfigureTask` or with the ddev-contrib add-on installed `ddev sake dev/tasks/SolrConfigureTask`
```

Comment:
`vendor/bin` is already in the $PATH inside the container. No big deal, but it makes it simpler to just use `ddev exec sake`

```suggestion
6. Run the Silverstripe Solr configuration command `ddev exec sake dev/tasks/SolrConfigureTask` or with the ddev-contrib add-on installed `ddev sake dev/tasks/SolrConfigureTask`
```